### PR TITLE
[Classic] Character Thumbnails

### PR DESCRIPTION
**Pull request by jazminite**
### Description
Add the ability to pull thumbnails from the Blizzard forum data.

### Test
I ran this locally with my updated WoWAnalyzer branch https://github.com/WoWAnalyzer/WoWAnalyzer/pull/6059

- Test report: `/report/FcfQtLmkXwpybJ1Y/27-Hardmode+Thorim+-+Kill+(2:27)`
- Screenshot:
image

- Test API url: `http://localhost:3001/i/character/classic/57875075/US/Benediction/Jazminites`
- Screenshot:
image

### CODE REVIEW

File: src/modules/character/index.js [hunk 1]
```diff
@@ -227,5 +311,18 @@ router.get('/i/character/:id([0-9]+)/:region([A-Z]{2})/:realm([^/]{2,})/:name([^
   // noinspection JSIgnoredPromiseFromCall
   fetchCharacter(region, realm, name, !responded ? res : null);
 });
+router.get('/i/character/classic/:id([0-9]+)/:region([A-Z]{2})/:realm([^/]{2,})/:name([^/]{2,})', cors, async (req, res) => {
+  const { id, region, realm, name } = req.params;
+  const storedCharacter = await getStoredCharacter(id);
```
Comment (emallson) [inline src/modules/character/index.js:316]: making a note for later that the id space for classic may overlap with retail, but even if it does the odds of it ever mattering is pretty low

File: src/helpers/BlizzardApi.js [hunk 2]
```diff
@@ -46,6 +46,14 @@ class BlizzardApi { // TODO: extends ExternalApi that provides a generic _fetch
     });
   }
 
+  async fetchClassicCharacter(regionCode, realm, name) {
```
Comment (emallson) [inline src/helpers/BlizzardApi.js:49]: quick explanatory comment on why the forum is used.

```suggestion
  /**
   * Blizzard does not expose an Armory for Classic characters, but they expose structured data from the forum about the character, including a thumbnail (which is what we mostly care about).
   *
   * This approach is taken from the method that WCL uses for claiming characters and collecting thumbnails for Classic.
   */
  async fetchClassicCharacter(regionCode, realm, name) {
```

**Review by emallson — APPROVED** [review]
personally would add the comment but code-wise this looks good

File: src/helpers/BlizzardApi.js (hunk 2 above)
Comment (jazminite) [reply src/helpers/BlizzardApi.js:54]: lgtm 👍

**Pull request merged**